fe/fe-core/src/main/java/com/starrocks/catalog/RangePartitionInfo.java

```diff
@@ -72,9 +72,9 @@ public class RangePartitionInfo extends PartitionInfo {
     @SerializedName(value = "partitionColumns")
     protected List<Column> partitionColumns = Lists.newArrayList();
     // formal partition id -> partition range
-    protected Map<Long, Range<PartitionKey>> idToRange = Maps.newHashMap();
+    protected Map<Long, Range<PartitionKey>> idToRange = Maps.newConcurrentMap();
```

Comment:
needn't concurrent?

Reply:
is unnecessary to use hash-map? and will set concurrentMap when rollback editlog now